scripts/india_udise/udise_schools_having_computer_available/README.md

```diff
@@ -0,0 +1,65 @@
+# UDISE+ - Number of Schools having Computer Available by School Category and Management 
+
+## About the Dataset
+This dataset gives the number of schools that have computer available. This dataset (Report Id:3109) has count of schools by school level and school management.
+
+Data is available at State, District, and Block levels. Currently, we are importing State and District level data.
+
+### Download URL
+A backend API allows the data to be queryable and responds with JSON data. The API endpoint is at `https://dashboard.udiseplus.gov.in/BackEnd-master/api/report/getTabularData`. The API takes JSON input with two main attributes.
+
+- dependencyValue - JSON value of query elements like year, state, dist, block
+- mapid - 80 - ID for the report
+
+#### API Output
+These are the attributes that we will use
+| Attribute       | Description                                                  |
+|--------|--------------------------------------------------------------|
+| cat1   | PS (I-V) - Primary only with grades 1 to 5                   |
+| cat2   | UPS (I-VIII) - Upper Primary with grades 1 to 8               |
+| cat3   |  HSS (I-XII) - HigherSecondary with grades 1 to 12           |
+| cat4   | UPS (VI-VIII) - Upper Primary only with grades 6 to 8        |
+| cat5   | HSS (VI-XII)    - Higher Secondary with grades 6 to 12       |
+| cat6   | SS (I-X) - Secondary/Sr. Sec.with grades 1 to 10             |
+| cat7   | SS (VI-X) - Secondary/Sr. Sec.with grades 6 to 10            |
+| cat8   | SS (IX-X) - Secondary/Sr. Sec.only with grades 9 & 10        |
+| cat10  | HSS (IX-XII) - Higher Secondary with grades 9 to 12          |
+| cat11  | HSS (XI-XII) - Hr.Sec. /Jr. College only with grades 11 & 12 |
+| total  | Total across all the school levels |
+| sch_mgmt_name  | School Management Type |
+
+
+#### Cleaned Data
+Cleaned data will be inside [UDISEIndia_Schools_Having_Computer_Available.zip](UDISEIndia_Schools_Having_Computer_Available.zip) as a CSV file with the following columns.
```

Comment:
The PR has UDISEIndia_Schools_Having_Computer_Available.csv that is 30MB. Is that supposed to be stored as a zip?

Reply:
Yes. Done.